datasm/resources/datasm_config.yaml

```diff
@@ -57,6 +57,6 @@ cmip_ocn_mon:
   num_workers: 12
   account: e3sm
   partition: debug
-  timeout: 36000
+  timeout: 43200
```

Comment:
should this be '43200' as other timeout setting?

Reply:
Believe it or not, some of the cwl_workflows were written to accept a string value, and others an integer value.  I learned the hard way that they are inconsistent.  They should be made consistent in the cwl workflow definitions, but I am not sufficiently familiar with that material to make the changes there.  As it stands, there are 3 opportunities to make adjustment.  (1) in datasm_config, (2) in the workflow/jobs/Generate* codes that gather and forge the inputs to cwltool, and (3) in the cwl_workflows.  If I make things uniform in the (1) datasm_config, I will need to add special handling in the (2) job setup codes.  Ridiculous?  Yes.